op.h

```diff
@@ -250,14 +250,14 @@ struct methop {
 
 struct pmop {
     BASEOP
+    U32         op_pmflags;
```

Comment:
A comment clarifying the reason for the placement here would be useful, especially for when BASEOP next changes.

Reply:
Something like this? 

`    U32         op_pmflags;     /* Situated to prevent x86-64 alignment hole */`

Reply:
This patch is wrong. The various OP structs are supposed to align where they share common fields (such as op_first) so that they can be cast between themselves. This commit is likely to misalign op_first in the PMOP struct on 32-bit builds, or possibly later on 64-bit builds if BASEOP changes.

Reply:
Ok, thanks Dave. There doesn't seem to be any other way to close these alignment holes, so I'll close this PR.